Formula/subversion.rb

```diff
@@ -235,21 +240,19 @@ def caveats
     system "#{bin}/svnadmin", "create", "test"
     system "#{bin}/svnadmin", "verify", "test"
 
-    if Hardware::CPU.intel?
-      platform = "darwin-thread-multi-2level"
-      on_linux do
-        platform = "x86_64-linux-thread-multi"
-      end
-
-      perl = "/usr/bin/perl"
-      on_linux do
-        perl = "#{Formula["perl"].opt_bin}/perl"
-      end
+    platform = "darwin-thread-multi-2level"
+    on_linux do
+      platform = "x86_64-linux-thread-multi"
+    end
```

Comment:
```suggestion
    platform = if OS.mac?
      "darwin-thread-multi-2level"
    else
      "#{Hardware::CPU.arch}-#{OS.kernel_name.downcase}-thread-multi"
    end
```

We've actually stopped using `on_macos` and `on_linux` blocks in `test`s now, so let's update this while we're here.

Reply:
Updated!  Good call with making "x86_64-linux" dynamic-- I should have searched the entire file for "x86_64" when I changed that other instance.